2024-05-24 11:35:26,209:INFO:Running command: python watermarked_text_generator.py ++attack_args.prompt_file='./inputs/mini_c4.csv' ++attack_args.prompt_num=1 ++watermark_args.save_file_name='c4_1.csv' ++attack_args.is_completion=True 
2024-05-24 11:35:26,209:INFO:Saving results to ./logs/c4/minic4_1_second.log
Traceback (most recent call last):
  File "/local1/borito1907/impossibility-watermark/generate_c4_texts.py", line 46, in <module>
    main()
  File "/local1/borito1907/impossibility-watermark/generate_c4_texts.py", line 36, in main
    stdout, stderr = run_command(command, log_filepath)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/local1/borito1907/impossibility-watermark/generate_c4_texts.py", line 15, in run_command
    result = subprocess.run(command, shell=True, check=True, text=True, stdout=f, stderr=subprocess.STDOUT)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/local1/borito1907/anaconda3/envs/watermark/lib/python3.12/subprocess.py", line 550, in run
    stdout, stderr = process.communicate(input, timeout=timeout)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/local1/borito1907/anaconda3/envs/watermark/lib/python3.12/subprocess.py", line 1201, in communicate
    self.wait()
  File "/local1/borito1907/anaconda3/envs/watermark/lib/python3.12/subprocess.py", line 1264, in wait
    return self._wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/local1/borito1907/anaconda3/envs/watermark/lib/python3.12/subprocess.py", line 2051, in _wait
    (pid, sts) = self._try_wait(0)
                 ^^^^^^^^^^^^^^^^^
  File "/local1/borito1907/anaconda3/envs/watermark/lib/python3.12/subprocess.py", line 2009, in _try_wait
    (pid, sts) = os.waitpid(self.pid, wait_flags)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
